### PR TITLE
Don't scan directories with .nomedia or .nomusic files

### PR DESCRIPTION
**Pull request by mildred**
Should solve issue #3584. I am currently testing it out on my large hard drive that contain music and many other things (that slows down the crawling)

### CODE REVIEW

File: src/library/librarywatcher.h [hunk 1]
```diff
@@ -26,6 +26,9 @@
 #include <QStringList>
 #include <QMap>
 
+#define LIBRARYWATCHER_NOMEDIA_FILE ".nomedia"
```
Comment (hatstand) [inline src/library/librarywatcher.h]: Put these in the cpp file and change them to:
```c++
namespace {
static const char* kNoMediaFile = ".nomedia";
...
```

**Comment by mildred** [comment]
By the way, it works really well on a slow computer with a hard disk containing lots of other files.

File: src/library/librarywatcher.cpp [hunk 2]
```diff
@@ -43,6 +43,11 @@
 #undef RemoveDirectory
 #endif
 
+namespace {
+static const char *kNoMoMediaFile = ".nomedia";
```
Comment (hatstand) [inline src/library/librarywatcher.cpp]: `kNoMediaFile`

**Comment by mildred** [comment]
I fixed my commit according to your comment

**Pull request merged**